### ISSUE TITLE
Is this a different QR format?

### ISSUE THREAD

**Issue by BarisGURENC** [admin-ch/CovidCertificate-App-Android#283]
We are testing the app with Samsung A71 Phone and Casper VIA S48 Tablet both.
But the tablet does not regognize the covid certificate qr code. But it does regognize other QR codes without any problem on QR Code Reader Software.

Here is the tablet's specifications : https://www.casper.com.tr/tablet/casper-via-s48

Tablet has 5mp rear, 2mp back camera and **regognize other qr codes without any problem**.

@goebelUB

**Comment by goebelUB** [comment]
Indeed, QR code recognition has always been one of the pain points. ZXing is not as reliable and "plug n play" as other solutions, but it is still one of the best FOSS libraries.

We are continuously tweaking our usage of ZXing to make it work on more devices and in scenarios. You can try v2.6.0 which includes more changes in this area. You can also try and debug it yourself and send a PR with improvements :)

> regognize other qr codes without any problem

1. With which app(s) does the tablet recognise other QR codes?
2. Does this app use ZXing? Is it FOSS and can we thus learn what it does differently?
3. Does this app recognise QR codes of the same size as DCC QR codes, or are the recognised QR codes smaller (and thus easier)?
